### PR TITLE
Closes #147 Notification Refactoring + Components Decomposition

### LINKED ISSUE #147: Notification Types Improval and Components separation

### PR DESCRIPTION
**Pull request by Reennon**

### CODE REVIEW

**Review by RamzesUAA — APPROVED** [review]

File: SoftServeCarMobile/src/activity/notifications/Notifications.tsx [hunk 1]
```diff
@@ -45,9 +44,16 @@ export default function Notifications(props: any) {
     return (
         <ScrollView style={NotificationStyle.headerContainer}>
             {notifications.map((item, key) => (
-                <>
-                    <NotificationComponent item={item} key={key} />
-                </>
+                /*<JourneyNewApplicant participant={
```
Comment (nrevutska) [inline SoftServeCarMobile/src/activity/notifications/Notifications.tsx:47]: Remove commented code

File: SoftServeCarMobile/src/components/journey-new-applicant/JourneyNewApplicant.tsx [hunk 2]
```diff
@@ -83,50 +98,54 @@ export const JourneyNewApplicant = () => {
...
-                                            style={
-                                                JourneyNewApplicantStyle.more
-                                            }
-                                        >
-                                            ...
-                                        </Text>
-                                    </TouchableOpacity>
-                                </View>
-                            </View>
+                            {/*<View style={item(10)}>*/}
```
Comment (nrevutska) [inline SoftServeCarMobile/src/components/journey-new-applicant/JourneyNewApplicant.tsx:101]: Remove commented code

File: SoftServeCarMobile/src/components/journey-new-applicant/JourneyNewApplicant.tsx [hunk 3]
```diff
@@ -83,50 +98,54 @@ export const JourneyNewApplicant = () => {
...
-                                The baggage is allowed in your car
-                            </Text>
+                            ]}>
+                            {props.participant!.hasLuggage ? (
+                                <Text
+                                    style={JourneyNewApplicantStyle.optionsHeader}
+                                >
+                                    I’m Traveling with a baggage.
+                                </Text>) : <View/>}
+                            {/*<Text style={JourneyNewApplicantStyle.optionsValue}>*/}
```
Comment (nrevutska) [inline SoftServeCarMobile/src/components/journey-new-applicant/JourneyNewApplicant.tsx:146]: Also, remove commented code here

File: SoftServeCarMobile/src/activity/notifications/NotificationComponent.tsx [hunk 4]
```diff
@@ -1,75 +1,27 @@
...
-            case NotificationType.HRMarketingMessage:
-                setModalTitle("HR NEWS");
-                setRequestType("marketing news");
-                break;
-
-            case NotificationType.HRMarketingSurvey:
-                setModalTitle("HR SURVEY");
-                setRequestType("marketing survey");
-                break;
+    let componentsEnumDict: IComponentsEnum<INotificationProps> = {
```
Comment (nrevutska) [inline SoftServeCarMobile/src/activity/notifications/NotificationComponent.tsx:19]: Name interfaces without I prefix - it is a convention in typescript.
Also, the suffix Dict is redundant in componentsEnumDict

File: SoftServeCarMobile/src/components/user-avatar/UserAvatar.tsx [hunk 5]
```diff
@@ -1,22 +1,30 @@
...
+import React, {useEffect, useState} from "react";
+import {container} from "tsyringe";
 import UserService from "../../../api-service/user-service/UserService";
-import AvatarInitials from "./AvatarInitials";
-import NotificationStyle from "./NotificationStyle";
+import NotificationStyle from "../../activity/notifications/NotificationStyle";
+import {AccentColors} from "../../common/enums/AccentColors";
 
-const AvatarComponent = (props: any) => {
+//<UserAvatar userId={64}
```
Comment (nrevutska) [inline SoftServeCarMobile/src/components/user-avatar/UserAvatar.tsx:9]: Please, remove this comment

**Review by NazariyChetvertukha — APPROVED** [review]

**Pull request merged**